### PR TITLE
Fixed #32698 -- Moved HttpRequest.get_raw_uri() to ExceptionReporter._get_raw_insecure_uri().

### PR DESCRIPTION
**Pull request by hramezani**
https://code.djangoproject.com/ticket/32698

### CODE REVIEW

File: django/http/request.py [hunk 1]
```diff
@@ -180,17 +180,6 @@ def get_signed_cookie(self, key, default=RAISE_ERROR, salt='', max_age=None):
                 raise
         return value
 
-    def get_raw_uri(self):
```
Comment (felixxm) [inline django/http/request.py:183]: Can you add a small release notes? e.g.
```
* The undocumented ``HttpRequest.get_raw_uri()`` method is removed.
```

Comment (hramezani) [reply django/http/request.py:183]: Sure. just added!

**Comment by adamchainz** [comment]
Swift work 👍

File: docs/releases/4.0.txt [hunk 2]
```diff
@@ -399,6 +399,8 @@ Miscellaneous
 * The undocumented ``django.contrib.admin.utils.lookup_needs_distinct()``
   function is renamed to ``lookup_spawns_duplicates()``.
 
+* The undocumented ``HttpRequest.get_raw_uri()`` method is removed.
```
Comment (adamchainz) [inline docs/releases/4.0.txt:402]: Maybe a mention of `build_absolute_uri` ?

Comment (felixxm) [reply docs/releases/4.0.txt]: Added.

**Pull request merged**